spring-boot-project/spring-boot-docs/src/docs/asciidoc/data/nosql.adoc

```diff
@@ -41,6 +41,41 @@ The following listing shows an example of such a bean:
 include::{docs-java}/features/nosql/redis/connecting/MyBean.java[]
 ----
 
+You can set the configprop:spring.data.redis.url[] property to change the URL and configure
```

Comment:
@Buzzardo would you happen to be available for some of your famously fussy feedback 😺 ?

ℹ️ I based this on the MongoDB URL in the section directly below it.